lib/ex_aws/s3.ex

```diff
@@ -1253,6 +1266,41 @@ defmodule ExAws.S3 do
     end
   end
 
+  @spec presigned_post(
+          config :: map,
+          bucket :: binary,
+          key :: binary | nil,
+          opts :: presigned_post_opts()
+        ) :: presigned_post_result()
+  def presigned_post(config, bucket, key, opts \\ []) do
+    expires_in = Keyword.get(opts, :expires_in, 3600)
+    {:ok, datetime} = DateTime.now("Etc/UTC")
+    expiration_date = DateTime.add(datetime, expires_in, :second)
+
+    datetime = datetime_to_erlang_time(datetime)
+
+    credential = ExAws.Auth.Credentials.generate_credential_v4("s3", config, datetime)
+
+    policy =
+      build_amz_post_policy(datetime, expiration_date, bucket, credential, opts, key)
+      |> config.json_codec.encode!
```

Comment:
Calling functions without parenthesis is no longer The Elixir Way - the compiler generally throws a warning about this, and only fails to here because it can't tell at compile time that it's actually a function (since the module name itself is a variable).